python/ray/tune/experiment/experiment.py

```diff
@@ -143,26 +145,57 @@ def __init__(
         export_formats: Optional[Sequence] = None,
         max_failures: int = 0,
         restore: Optional[str] = None,
+        # Deprecated
+        local_dir: Optional[str] = None,
     ):
+        if isinstance(sync_config, dict):
+            sync_config = SyncConfig(**sync_config)
+        else:
+            sync_config = sync_config or SyncConfig()
+
+        self.sync_config = sync_config
+
+        local_storage_path = storage_path
```

Comment:
ohhhh so `local_storage_path` is the parent folder of `local_path`?

Reply:
Yes that's right. The `local_storage_path` rename is just for clarity in the code (spoiler: it will be relevant when we move to only passing one `storage_path`)